src/helpers/SellerHelper.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * HiPanel core package
+ *
+ * @link      https://hipanel.com/
+ * @package   hipanel-core
+ * @license   BSD-3-Clause
+ * @copyright Copyright (c) 2014-2019, HiQDev (http://hiqdev.com/)
+ */
+
+namespace hipanel\helpers;
+
+use Yii;
+use yii\base\InvalidConfigException;
+
+class SellerHelper
+{
+    public static function get() : string
```

Comment:
```suggestion
    public static function getLogin(): string
```

Как насчёт более явного имени?

Reply:
поддерживаю